src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs

```diff
@@ -68,19 +68,19 @@ internal static class CsvAppFileIO {
             Interlocked.Exchange(ref _busy, 0);
         }
 
-        private static async Task CreateCsvAndStartProcess(IREvaluationResultInfo result, IRSession session, string file) {
+        private static async Task CreateCsvAndStartProcess(IREvaluationResultInfo result, IRSession session, string fileName) {
             await TaskUtilities.SwitchToBackgroundThread();
 
             var sep = CultureInfo.CurrentCulture.TextInfo.ListSeparator;
             var dec = CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator;
 
             using (var e = await session.BeginEvaluationAsync()) {
-                var csvdata = await e.EvaluateAsync($"rtvs:::export_to_csv({result.Expression}, sep={sep.ToRStringLiteral()}, dec={dec.ToRStringLiteral()})", REvaluationKind.RawResult);
-                csvdata.SaveRawDataToFile(file);
+                var csvData = await e.EvaluateAsync<byte[]>($"rtvs:::export_to_csv({result.Expression}, sep={sep.ToRStringLiteral()}, dec={dec.ToRStringLiteral()})", REvaluationKind.Normal);
+                File.WriteAllBytes(fileName, csvData);
```

Comment:
This entire class needs to be refactored to do that (probably by making it non-static, and taking `IFileSystem` etc in constructor). I'll file an issue.